packages/dialtone-vue2/components/hovercard/hovercard.vue

```diff
@@ -0,0 +1,201 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<template>
+  <dt-popover
+    :id="id"
+    :placement="placement"
+    :content-class="contentClass"
+    :fallback-placements="fallbackPlacements"
+    :padding="padding"
+    :transition="transition ? 'fade' : null"
+    :offset="offset"
+    :modal="false"
+    initial-focus-element="none"
+    :header-class="headerClass"
+    :footer-class="footerClass"
+    :append-to="appendTo"
+    :hovercard="true"
+    :timer="timer"
+    data-qa="dt-hovercard"
+    @opened="(e) => ($emit('opened', e))"
+  >
+    <template #anchor="{ attrs }">
+      <slot
+        name="anchor"
+        v-bind="attrs"
+      />
+    </template>
+    <template #content>
+      <slot name="content" />
+    </template>
+    <template #headerContent>
+      <slot name="headerContent" />
+    </template>
+
+    <template #footerContent>
+      <slot name="footerContent" />
+    </template>
+  </dt-popover>
+</template>
+
+<script>
+import { reactive } from 'vue';
+import { POPOVER_APPEND_TO_VALUES, POPOVER_PADDING_CLASSES, DtPopover } from '@/components/popover/index.js';
+import { getUniqueString } from '@/common/utils';
+import useTimer from './timer';
+
+export default {
+  name: 'DtHovercard',
+
+  components: {
+    DtPopover,
+  },
+
+  props: {
+  /**
+   * Fade transition when the content display is toggled.
+   * @type boolean
+   * @values true, false
+   */
+    transition: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+   * If the popover does not fit in the direction described by "placement",
+   * it will attempt to change its direction to the "fallbackPlacements".
+   * <a
+   *   class="d-link"
+   *   href="https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements"
+   *   target="_blank"
+   * >
+   *   Popper.js docs
+   * </a>
+   * */
+    fallbackPlacements: {
+      type: Array,
+      default: () => {
+        return ['auto'];
+      },
+    },
+
+    /**
+   * The direction the popover displays relative to the anchor.
+   * <a
+   *   class="d-link"
+   *   href="https://atomiks.github.io/tippyjs/v6/all-props/#placement"
+   *   target="_blank"
+   * >
+   *   Tippy.js docs
+   * </a>
+   * @values top, top-start, top-end,
+   * right, right-start, right-end,
+   * left, left-start, left-end,
+   * bottom, bottom-start, bottom-end,
+   * auto, auto-start, auto-end
+   */
+    placement: {
+      type: String,
+      default: 'top-start',
+    },
```

Comment:
Since we have validator on padding prop, we could have a validator here, up to you

Reply:
Fixed in latest changes